docs/src/content/docs/book/structs-and-messages.mdx

```diff
@@ -109,6 +111,12 @@ message Add {
 
 Messages are almost the same as [Structs](#structs), with the only difference being that Messages have a 32-bit integer header in their serialization containing their unique numeric id, commonly referred to as an _opcode_ (operation code). This allows Messages to be used with [receivers](/book/receive), since the contract can distinguish different types of messages based on this id.
 
+Additionally and unlike regular [structs](#structs), this allows Messages to be declared empty:
```

Comment:
I would also add it's an idiom, since you can specify opcodes explicitly:

```tact
message(22) CloseMinting { }
``` 

```tact
message(3) ChangeOwner { }
```